content/building/build-versioning.md

```diff
@@ -103,6 +103,7 @@ Add the following script under your `scripts` field for `codemagic.yaml`, or as
 ```bash
 export APP_STORE_CONNECT_PRIVATE_KEY=$(echo $APP_STORE_CONNECT_PRIVATE_KEY | base64 --decode) # if you encrypted the file itself, not its content
 LATEST_BUILD_NUMBER=$(app-store-connect get-latest-app-store-build-number '1234567890') # The argument is your application's Apple ID
+cd $FCI_BUILD_DIR/ios
```

Comment:
Since default working directory is anyway `$FCI_BUILD_DIR`, I find that just `cd ./ios` would be more appropriate as otherwise there could be some confusion on what does the used variable exactly mean.